src/profile.c

```diff
@@ -259,13 +260,19 @@ void profile_update_leds() {
     }
 }
 
-void profile_report_active() {
-    if (profile_pending_reboot && !home_is_active) config_reboot();
+// Check if profile has been requested to be reset (usually after profile change).
+void profile_check_reset() {
```

Comment:
THis is not checking if there is / or should be a reset, but it is executing a reset